<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sonivox EAS Synth</string>
  </property>
  <property name="windowIcon">
   <iconset resource="guisynth.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblMIDIFile">
        <property name="text">
         <string>MIDI File:</string>
        </property>
        <property name="buddy">
         <cstring>playButton</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="playButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="guisynth.qrc">
            <normaloff>:/play.png</normaloff>:/play.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="stopButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="guisynth.qrc">
            <normaloff>:/stop.png</normaloff>:/stop.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="openButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="guisynth.qrc">
            <normaloff>:/open.png</normaloff>:/open.png</iconset>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblSong">
          <property name="text">
           <string>[empty]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblMIDIIn">
        <property name="text">
         <string>MIDI IN:</string>
        </property>
        <property name="buddy">
         <cstring>combo_MIDI</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="combo_MIDI"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblAudioOut">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Audio Output:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>combo_Device</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="combo_Device">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="volumeLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Volume:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>volumeSlider</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSlider" name="volumeSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>90</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblBufferTime">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Buffer Time:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>spinBuffer</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="spinBuffer">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="suffix">
           <string> ms</string>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblOctave">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Octave:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinOctave">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>8</number>
          </property>
          <property name="value">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="drumstick::widgets::PianoKeybd" name="pianoKeybd">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>130</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>450</width>
        <height>130</height>
       </size>
      </property>
      <property name="baseOctave">
       <number>3</number>
      </property>
      <property name="numKeys">
       <number>25</number>
      </property>
      <property name="keyPressedColor">
       <color>
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>8</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lblReverb">
      <property name="text">
       <string>Reverb</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="lblChorus">
      <property name="text">
       <string>Chorus</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QDial" name="dial_Reverb">
      <property name="maximum">
       <number>32765</number>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QDial" name="dial_Chorus">
      <property name="maximum">
       <number>32765</number>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QComboBox" name="combo_Reverb"/>
    </item>
    <item row="5" column="2">
     <widget class="QComboBox" name="combo_Chorus"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>drumstick::widgets::PianoKeybd</class>
   <extends>QGraphicsView</extends>
   <header location="global">drumstick/pianokeybd.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="guisynth.qrc"/>
 </resources>
 <connections/>
</ui>
